runners_manager/runner/RunnerManager.py

```diff
@@ -65,6 +65,13 @@ def update(self, github_runners: list[dict]):
             while self.need_new_runner(vm_type):
                 self.create_runner(vm_type)
 
+            # Respawn runner if they are offline for more then 10min after spawn
```

Comment:
Your comment says 10 minutes, the code says 15.
```suggestion
            # Respawn runner if they are offline for more then 15min after spawn
```